app/views/recipes/_recipe.html.erb

```diff
@@ -0,0 +1,83 @@
+<div id="<%= dom_id recipe %>">
+  <p>
+    <h2 class="title is-2"><%= recipe.name %></h2>
+  </p>
+  <div class="box">
+    <article class="media">
+      <div class="media-content">
+        <div class="content">
+          <p>
+            <strong>Preparation time:</strong>
+            <%= recipe.preparation_time %>
+          </p>
+          <p>
+            <strong>Cooking time:</strong>
+            <%= recipe.cooking_time %>
+          </p>
+          <p><strong>Steps go here...</strong></p>
+        </div>
+        <br />
+        <nav class="level">
+          <div class="level-left is-mobile">
+              <%= link_to recipe, class: "level-item" do %>
+                <button class="button is-save is-outlined is-rounded is-responsive">Generate shopping list</button>
+              <% end %>
```

Comment:
- [ ] There seems to be a bug with the generate shopping list functionality as it doesn't work. Pressing the button to generate a shopping list simply does nothing. Please implement this functionality 🙏

Reply:
@codepantha this functionality is for 3 people group